aiida/orm/implementation/querybuilder.py

```diff
@@ -126,6 +127,52 @@ def get_filter_expr_from_attributes(cls, operator, value, attr_key, column=None,
         """
         pass
 
+    @classmethod
+    def get_filter_expr_from_column(cls, operator, value, column):
```

Comment:
I notice this has SqlAlchemy imports but is moved to the abstract class. Is this intentional? Because of how the querybuilder for django actually also operates on SqlAlchemy in some way?